dev-server/static/scripts/pdfjs-init.js

```diff
@@ -1,62 +1,21 @@
 'use strict';
 
-// Note: This file is not transpiled.
+/* global PDFViewerApplication, PDFViewerApplicationOptions */
```

Comment:
It is not clear why you use `/* */` format for this comment.
```suggestion
// PDFViewerApplication and PDFViewerApplicationOptions are defined globally
```

Reply:
This comment is for ESLint. See https://eslint.org/docs/rules/no-undef. It requires the use of `/* ... */`.

Reply:
I asked because for some reason `eslint` doesn't complain if `/* global .... */` is removed.